### PR TITLE
543 update tests

### PR DESCRIPTION
**Pull request by almasaeed2010**
## Type(s) of Change(s)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] API-specific change (fix or addition to an API function)
- [x] Updates documentation (inline or markdown files)

## Description

This PR:
- Provides a test for the method that was failing in issue #539 amongst other new tripal_job tests.
- Upgrades tripal-test-suite and adds an example of how to use the `silent` method in tests.
- Utilizes tripal test suite's `silent` method where appropriate instead of `ob_start()` and `ob_end_clean()`

## Testing?

1. Checkout the branch and run phpunit (or see the log in travis ci).

## Additional Notes (if any):

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by bradfordcondon** [comment]
THank you @almasaeed2010 !
```
phpunit
PHPUnit 6.5.2 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.1.16
Configuration: /Users/chet/UTK/tripal/sites/all/modules/tripal/phpunit.xml

.......................                                           23 / 23 (100%)

Time: 1.86 minutes, Memory: 48.00MB

OK (23 tests, 47 assertions)
```

tests run fine for me.    Considering this is just bumping the version, and we added a note to the README about updating composer (since I know some of our devs dont necessarily use package managers in their drupal work) we should be good to go.

Once CI passes I'll merge.

File: tests/tripal/api/TripalJobsApiTest.php [hunk 1]
```diff
@@ -13,8 +14,49 @@ class TripalJobsApiTest extends TripalTestCase {
...
   }
+
+  /** @test */
+  public function testRetrievingAJob() {
+    $job = factory('tripal_jobs')->create();
+
+    $job2 = tripal_get_job($job->job_id);
+
+    $this->assertNotEmpty($job2);
+    $this->assertObjectHasAttribute('job_id', $job2);
```
Comment (laceysanderson) [inline tests/tripal/api/TripalJobsApiTest.php:30]: Maybe add `$this->assertEquals($job2->job_id, $job->job_id)` as an easy test that it retrieved the right job?

**Review by laceysanderson — APPROVED** [review]
Looks Awesome! Thanks for contributing more tests!!! <3

File: tests/tripal/api/TripalJobsApiTest.php (hunk 1 above)
Comment (almasaeed2010) [reply tests/tripal/api/TripalJobsApiTest.php:30]: Added. Thanks for the suggestion!

**Comment by laceysanderson** [comment]
Looks like tests passed in Travis so I'll make the merge since I am in complete agreement with @bradfordcondon :-) Thanks @almasaeed2010!!!

**Pull request merged**